libraries/splunk_password.rb

```diff
@@ -0,0 +1,99 @@
+# coding: UTF-8
+#
+# Cookbook Name:: cerner_splunk
+# File Name:: splunk_password.rb
+
+require 'base64'
+
+# Module contains different functions to encrypt and decrypt splunk passwords
+module CernerSplunk
+  # RC4 implementation taken from https://github.com/caiges/Ruby-RC4
```

Comment:
Charlie suggested moving the RC4 class to a separate file to avoid confusions around Licenses.
51836556ce605c65d15c25f55f9093fbbe972958